crates/node/src/roles/admin.rs

```diff
@@ -58,13 +58,8 @@ impl AdminRole {
     ) -> Result<Self, AdminRoleBuildError> {
         let config = updateable_config.pinned();
 
-        // Total duration roughly 66 seconds
-        let retry_policy = RetryPolicy::exponential(
-            Duration::from_millis(100),
-            2.0,
-            Some(10),
-            Some(Duration::from_secs(20)),
-        );
+        // Total duration roughly 1s
+        let retry_policy = RetryPolicy::exponential(Duration::from_millis(100), 2.0, Some(4), None);
```

Comment:
How long does it take to spin up a cold Lambda?

Reply:
this duration is unrelated to the timeout on the request to the lambda - its just the duration between retries. in the lambda case the first request will block on the cold start, and then likely succeed. if it somehow fails transiently, an immediate subsequent retry will most likely *not* see a cold start, and then succeed immediately. in no scenario would a super slow cold start lead us to breach this retry policy

Reply:
True, it is the duration between failures. Thanks for the clarification.